cmd/zap/text_test.go

```diff
@@ -215,10 +215,58 @@ func TestExpander(t *testing.T) {
 		var res bytes.Buffer
 		res.WriteString(httpsPrefix)
 
-		expandPath(c, l.Front(), &res)
+		ExpandPath(c, l.Front(), &res)
 
 		Convey("result should equal 'https://kafka.apache.org/expand/javadoc/index.html?overview-summary.html", func() {
 			So(res.String(), ShouldEqual, "https://kafka.apache.org/expand/javadoc/index.html?overview-summary.html")
 		})
 	})
+	Convey("Given 'g/s/me'", t, func() {
+		c, _ := loadTestYaml()
+		l := tokenize("g/s/me")
+		var res bytes.Buffer
+		res.WriteString(httpsPrefix)
+
+		ExpandPath(c, l.Front(), &res)
+
+		Convey("result should equal 'https://github.com/search?q=issmirnov'", func() {
+			So(res.String(), ShouldEqual, "https://github.com/search?q=issmirnov")
+		})
+	})
+	Convey("Given 'g/s/me.z'", t, func() {
```

Comment:
I'm guessing this is a typo - did you mean to put a slash instead between `me` and `z` ?

Reply:
Thanks, fixed 👍